includes/definitions/bintec-beip-plus.yaml

```diff
@@ -0,0 +1,14 @@
+os: bintec-beip-plus
+nobulk: true
```

Comment:
This is now snmp_bulk. You should rebase this pr

Reply:
Done 👍